backend/migrations/00071_add_facility_feature_flags.sql

```
@@ -0,0 +1,27 @@
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    PRIMARY KEY (facility_id, feature),
+    FOREIGN KEY (facility_id) REFERENCES public.facilities(id) ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+CREATE INDEX idx_facility_feature_flags_feature ON public.facility_feature_flags USING btree (feature);
+
+-- Per-facility control now supersedes the statewide staged-rollout gate that
+-- kept this off by default; flip the floor to enabled so it behaves like the
+-- other three top-level features (on by default, opt out per facility).
+UPDATE public.feature_flags SET enabled = TRUE WHERE name = 'learning_record';
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)